app/V1Module/presenters/EmailsPresenter.php

```diff
@@ -100,4 +115,34 @@ public function actionSendToRegularUsers() {
     $this->emailHelper->send(null, [], $subject, $message, $emails);
     $this->sendSuccessResponse("OK");
   }
+
+  public function checkSendToGroupMembers(string $groupId) {
+    $group = $this->groups->findOrThrow($groupId);
+    if (!$this->groupAcl->canSendEmail($group)) {
+      throw new ForbiddenRequestException();
+    }
+  }
+
+  /**
+   * Sends an email with provided subject and message to all members of given group.
+   * @POST
+   * @param string $groupId
+   * @Param(type="post", name="subject", validation="string:1..", description="Subject for the soon to be sent email")
+   * @Param(type="post", name="message", validation="string:1..", description="Message which will be sent, can be html code")
+   * @throws NotFoundException
+   */
+  public function actionSendToGroupMembers(string $groupId) {
```

Comment:
Do we really want to send the message to supervisors and admins too? I could imagine a flag here.

Reply:
Yes, that is maybe better solution to have one action for a group and have switches for the particular user membership groups.

Reply:
I would like to request one more flag here: It would be nice to have "I want a copy of that email" option. It may be useful when I want a copy, but I do not want to notify all admins or when I am a superadmin managing group that is not mine.

Reply:
Fair enough... Can do